bin/start

```diff
@@ -101,6 +101,7 @@ fi
 JAVA_OPTS="$JAVA_OPTS -XX:+IgnoreUnrecognizedVMOptions" # Don't barf if we see an option we don't understand (e.g. Java 9 option on Java 7/8)
 JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true"         # don't try to start AWT. Not sure this does anything but better safe than wasting memory
 JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"            # Use UTF-8
+JAVA_OPTS="$JAVA_OPTS -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager" # java.util.logging via log4j: https://logging.apache.org/log4j/2.x/log4j-jul/
```

Comment:
I'm not a huge fan of this since it doesn't work if people just do `java -jar metabase.jar`... can we set this at runtime instead?